### PR TITLE
Add Anorm license

### PR DESCRIPTION
**Pull request by flegall**

### CODE REVIEW

File: src/main/resources/license-data.json [hunk 1]
```diff
@@ -1907,6 +1907,10 @@
         "org.apache.maven:maven-builder-support": {
             "license": "Apache-2.0",
             "licenseSource": "data"
+        },
+        "com.typesafe.play:anorm_2.11": {
+            "license": "Apache-2.0",
+            "licenseSource": "data"
```
Comment (rklaren) [inline src/main/resources/license-data.json]: Please put it near the other typesafe stuff. And not make it a random sorted.

Comment (flegall) [inline src/main/resources/license-data.json]: Right

Comment (flegall) [inline src/main/resources/license-data.json]: Fixed

**Comment by flegall** [comment]
Thanks, can you release a new version as well ?

**Pull request merged**